Fix FTBFS with llvm < 22
lang++: error: unknown argument: '-fsanitize-ignore-for-ubsan-feature=array-bounds'

diff -up chromium-146.0.7680.65/build/config/compiler/BUILD.gn.than chromium-146.0.7680.65/build/config/compiler/BUILD.gn
--- chromium-146.0.7680.65/build/config/compiler/BUILD.gn.than	2026-03-09 11:10:29.942718861 +0100
+++ chromium-146.0.7680.65/build/config/compiler/BUILD.gn	2026-03-09 11:11:28.703554766 +0100
@@ -1855,18 +1855,6 @@ config("sanitize_c_array_bounds") {
     cflags = [
       "-fsanitize=array-bounds",
       "-fsanitize-trap=array-bounds",
-
-      # Some code users feature detection to determine if UBSAN (or any
-      # sanitizer) is enabled, they then do expensive debug like operations. We
-      # want to suppress this behaviour since we want to keep performance costs
-      # as low as possible while having these checks.
-      "-fsanitize-ignore-for-ubsan-feature=array-bounds",
-
-      # Because we've enabled array-bounds sanitizing we also want to suppress
-      # the related warning about "unsafe-buffer-usage-in-static-sized-array",
-      # since we know that the array bounds sanitizing will catch any out-of-
-      # bounds accesses.
-      "-Wno-unsafe-buffer-usage-in-static-sized-array",
     ]
   }
 }
